src/analyzing/AST_to_IL.ml

```diff
@@ -1108,6 +1108,11 @@ and stmt_expr env ?e_gen st =
   | G.Return (t, eorig, _) ->
       mk_s (Return (t, expr_opt env eorig)) |> add_stmt env;
       expr_opt env None
+  | G.DefStmt (ent, G.VarDef { G.vinit = Some e; vtype = _typTODO }) ->
```

Comment:
Should we guard by language as @ihji suggested here https://github.com/semgrep/semgrep/pull/9185#issuecomment-1811046530 ?

Reply:
Will do.